compiler/fm/src/file_reader.rs

```diff
@@ -33,7 +35,7 @@ cfg_if::cfg_if! {
 
         }
 
-        pub(crate) fn read_file_to_string(path_to_file: &Path) -> Result<String, Error> {
+        pub fn read_file_to_string(path_to_file: &Path) -> Result<String, Error> {
```

Comment:
Could we move the code in `if #[cfg(target_arch = "wasm32")] {` to compiler/wasm?

Reply:
Yeah, that's the plan. I just wanted to sound out the idea before doing the work.

Reply:
I think moving the file system impurity more towards `compiler/wasm` looks okay